collectors/logsmanagement.plugin/plugin_logsmanagement.c

```diff
@@ -0,0 +1,411 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#include "plugin_logsmanagement.h"
+
+#define WORKER_JOB_COLLECT 0
+#define WORKER_JOB_UPDATE  1
+
+static struct Chart_meta chart_types[] = {
```

Comment:
If you take a look in our overall code, you will see that we have some `define` to create upper case data type like [these](https://github.com/netdata/netdata/blob/185f5a183b423c4fb76bbcf70778f66e69c31fc1/database/rrd.h#L12-L31), but the datatype itself is in lower case. I can see in the beginning of this file that you are starting data type with Upper case. I suggest to keep ancient `C`  pattern and use everything in lower case and use upper case like we are using in database.

I understand we are using data type like this with `ML`, but they are C++ code.